code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -252,6 +252,19 @@ GLOBAL_LIST_INIT(cardboard_recipes, list ( \
 /obj/item/stack/sheet/cardboard/fifty
 	amount = 50
 
+/obj/item/stack/sheet/cardboard/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/stamp/clown))
+		return ..()
+	if(get_amount() >= 2)
+		to_chat(user, "<span class ='notice'>You only need one cardboard sheet for this.</span>")
```

Comment:
The `use()` proc handles this kind of thing with sheets. Use it in an if() statement. `use(5)` would use five items in a stack, deleting the stack if exactly 5 were remaining, for example. You could do something similar here.

Reply:
Yeah, but I've found a bug if you held the cardboard in your hands while having more than the use().
It use all the stack to make a box, and when you unfold, it returns to the stack number.

Reply:
I'll give it a try.

Reply:
Its a bit more complex than anticipated, give me some time.

Reply:
That took a while, but I did it.